src/QMCWaveFunctions/tests/test_TrialWaveFunction_diamondC_2x1x1.cpp

```diff
@@ -44,7 +44,7 @@ struct float_tag
  *  used and a constexpr based on DeterminantTypes that actually depend on legacy cuda.
  */
 template<class DiracDet, class SPO_precision>
-void testTrialWaveFunction_diamondC_2x1x1(const int ndelay)
+void testTrialWaveFunction_diamondC_2x1x1(const int ndelay, const bool offload_spo, const bool offload_jas)
```

Comment:
a struct for input would make this way more readable later. 
```
struct offload_comp {
bool spo;
bool jas
};

Reply:
Changed.